run-make-check.sh

```diff
@@ -65,19 +65,10 @@ function run() {
 	$DRY_RUN ./install-deps.sh || return 1
     fi
     export TMPDIR=$(mktemp -d --tmpdir ceph.XXX)
-    if test -x ./do_cmake.sh ; then
-        $DRY_RUN ./do_cmake.sh $@ || return 1
-        cd build
-        $DRY_RUN make $BUILD_MAKEOPTS tests || return 1
-        $DRY_RUN ctest $CHECK_MAKEOPTS --output-on-failure || return 1
-    else
-        $DRY_RUN ./autogen.sh || return 1
-        $DRY_RUN ./configure "$@"  --with-librocksdb-static --disable-static --with-radosgw --with-debug --without-lttng \
-            CC="ccache gcc" CXX="ccache g++" CFLAGS="-Wall -g" CXXFLAGS="-Wall -g" || return 1
-        $DRY_RUN make $BUILD_MAKEOPTS || return 1
-        $DRY_RUN make $CHECK_MAKEOPTS check || return 1
-        $DRY_RUN make dist || return 1
-    fi
+    $DRY_RUN ./do_cmake.sh $@ || return 1
```

Comment:
or we can just remove this script, and rename `run-cmake-check.sh` to `run-make-check.sh`?